apps/drec-ui/src/utils/formatters/PowerFormatter.ts

```diff
@@ -22,4 +22,11 @@ export class PowerFormatter {
             includeDisplayUnit ? ' ' + PowerFormatter.displayUnit : ''
         }`;
     }
+
+    static formatDisplay(powerInWatt: number, includeDisplayUnit?: boolean): string {
+        // @ts-ignore
```

Comment:
What was the error?

Reply:
Actually I removed this. It was from the method above which complains about this Unit[PowerFormatter.displayUnit] being any.